### PR TITLE
Fix largeBold-font vertical alignment

### PR DESCRIPTION
**Pull request by rottabonus**
###  Updated 12.1.2021: 

- MessageButton text was not vertically aligned correctly in the settings-form Logout and Delete Account -buttons. 
- After doing this and that I found out that the largeBold-fonts lineHeight caused the issue. Changed the lineHeight to 27 which made it look vertically aligned in both ios and android.

### CODE REVIEW

**Review by pheis — DISMISSED** [review]
Is buttonContainer still in use? Good change!

**Comment by ahto** [comment]
yeah very nice. 

is it just my eyes wonky or is the ios buttons now little different.

before this: android text was little below center. ios was in the center.
after: android text is in the center. ios is little above center.

**Comment by rottabonus** [comment]
Yeah, ButtonContainer is still used in two components. 

And about the alignment in ios..  
In the last pic (filter-screen) the text in Show-button does not seem aligned correctly. 
I'll double check it.

What do you think about ios/android specific styling?

**Comment by pheis** [comment]
> Yeah, ButtonContainer is still used in two components.
> 
> And about the alignment in ios.. In the last pic (filter-screen) the text in Show-button does not seem aligned correctly. I'll double check it.
> 
> What do you think about ios/android specific styling?

IMO better to use styleprops that behave consistently on both platforms, if not possible then we probably have to do some specific styling.

When specific stuff is done it would be nice to abstract it away to some component that is used in many places

**Comment by pheis** [comment]
It is highly probable that somehting goes wrong at some point when we have to code differently for the platforms people are always lazy to test on both platforms.

**Review by ahto — APPROVED** [review]
nice. i can cancel my silmälääkäri

File: src/Screens/components/fonts.ts [hunk 1]
```diff
@@ -42,12 +42,12 @@ const fonts = {
...
     fontFamily: 'Nunito-Regular',
     fontSize: 20,
-    lineHeight: 28,
+    lineHeight: 27,
   },
   largeBold: {
     fontFamily: 'Nunito-Bold',
     fontSize: 20,
-    lineHeight: 28,
+    lineHeight: 27,
```
Comment (pheis) [inline src/Screens/components/fonts.ts:50]: heh

**Review by pheis — APPROVED** [review]

**Pull request merged**